### PR TITLE
Fix constants wording

### PR DESCRIPTION
**Pull request by Flanker32**
Fix constants wording in #917

### CODE REVIEW

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java [hunk 1]
```diff
@@ -10,9 +10,9 @@
     public static final String APP_SETTING_WEBSITE_RUN_FROM_PACKAGE = "WEBSITE_RUN_FROM_PACKAGE";
     public static final String APP_SETTING_MACHINEKEY_DECRYPTION_KEY = "MACHINEKEY_DecryptionKey";
     public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
-    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE_VALUE = "false";
+    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE_FALSE = "false";
```
Comment (andxu) [inline azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java]: APP_SETTING_WEBSITES_DISABLE_APP_SERVICE_STORAGE

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java [hunk 2]
```diff
@@ -18,10 +18,10 @@
 import static com.microsoft.azure.maven.Utils.assureServerExist;
 import static com.microsoft.azure.maven.appservice.DockerImageType.PUBLIC_DOCKER_HUB;
 import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE;
-import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE_VALUE;
+import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE_READONLY;
```
Comment (andxu) [inline azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java]: use star for many constants

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java (hunk 1 above)
Comment (Flanker32) [reply azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java]: Will fix it in later PR

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java (hunk 2 above)
Comment (Flanker32) [reply azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java]: Will fix it in later PR

Comment (Flanker32) [reply azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java]: We enable the `AvoidStarImport` in checkstlye, which will block star imports.

**Comment by Flanker32** [comment]
Close and make a new pr

**Pull request closed without merging**